lib/jsonpatch/json-patch-duplex.js

```diff
@@ -330,6 +330,10 @@ var jsonpatch;
             generate(observer);
             clearTimeout(observer.next);
             removeObserverFromMirror(mirror, observer);
+            if (mirror.observers.length === 0)
+            {
```

Comment:
Incorrect format - braces must be on this same line after the preceding closing brace.